service/publish/handler_test.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"bou.ke/monkey"
+	"context"
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"io"
+	"os"
+	"path"
+	"reflect"
+	"regexp"
+	"runtime"
+	"testing"
+	"toktik/kitex_gen/douyin/publish"
+	"toktik/repo"
+	"toktik/storage"
+)
+
+var testVideo []byte
+
+func TestMain(m *testing.M) {
+	var err error
+	_, currentFilePath, _, _ := runtime.Caller(0)
+
+	testVideo, err = os.ReadFile(path.Join(path.Dir(currentFilePath), "resources/bear.mp4"))
```

Comment:
Consider enabling LFS for test resources.

Reply:
I don't think we need LFS, as long as we keep the files small and infrequently modified.

Reply:
Sure